app/request_handler.py

```diff
@@ -102,12 +102,15 @@ def readCachedVenueIdentifiers(cache):
         return cache.get("identifiers", None)
     return None
 
-def researchPlace(keyID, providersList, identifiers):
-    # TODO: Check more than just Proxwalk
-    placeProviderIDs = proxwalk.getAndCacheProviderIDs(keyID, providersList, identifiers)
-    venueDetails = search.getVenueDetails(placeProviderIDs)
-    writeVenueProviderRecord(keyID, venueDetails)
-    return [key.encode('utf-8') for key in venueDetails.keys()]
+def researchPlace(keyID, placeProviderIDs):
+    try:
+        venueDetails = search.getVenueDetails(placeProviderIDs)
+        writeVenueProviderRecord(keyID, venueDetails)
+        return [key.encode('utf-8') for key in venueDetails.keys()]
```

Comment:
py3: `unicode.encode` creates a byte string (which is probably not what we want here). I didn't fix it in my initial conversion but you could fix it here.